<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Route_Tool</class>
 <widget class="QDialog" name="Route_Tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Route Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon_resources.qrc">
    <normaloff>:/Menu Icon/Route_Tool.svg</normaloff>:/Menu Icon/Route_Tool.svg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QLabel" name="Route_View">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>25</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>30</y>
     <width>300</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Route Tool</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>550</y>
     <width>300</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Coordinate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>570</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(0,0)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>583</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Coordinate</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>317</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="16" column="1">
     <widget class="QPushButton" name="pushButton_view">
      <property name="locale">
       <locale language="English" country="UnitedKingdom"/>
      </property>
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>t</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_centreX">
      <property name="decimals">
       <number>9</number>
      </property>
      <property name="minimum">
       <double>-99.999999000000003</double>
      </property>
      <property name="maximum">
       <double>99.999999000000003</double>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_axisColour">
      <item>
       <property name="text">
        <string>Transparent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>White</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gray</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_t">
      <property name="decimals">
       <number>9</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_size">
      <property name="decimals">
       <number>9</number>
      </property>
      <property name="minimum">
       <double>-99.999999000000003</double>
      </property>
      <property name="maximum">
       <double>99.999999000000003</double>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Centre Coordinate (X,Y)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Axis Colour</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QSlider" name="Slider_time">
      <property name="maximum">
       <number>1000000000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_centreY">
      <property name="decimals">
       <number>9</number>
      </property>
      <property name="minimum">
       <double>-99.999999000000003</double>
      </property>
      <property name="maximum">
       <double>99.999999000000003</double>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="pushButton_Save">
      <property name="text">
       <string>Save Point</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Angle</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_angle">
      <property name="suffix">
       <string>°</string>
      </property>
      <property name="decimals">
       <number>9</number>
      </property>
      <property name="minimum">
       <double>-10000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="Axis_x">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>324</y>
     <width>600</width>
     <height>2</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="Axis_y">
   <property name="geometry">
    <rect>
     <x>649</x>
     <y>25</y>
     <width>2</width>
     <height>600</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions(&amp;A)</string>
    </property>
    <addaction name="actionView"/>
    <addaction name="actionSave_Point"/>
    <addaction name="separator"/>
    <addaction name="actionBack_to_main_window"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings(&amp;S)</string>
    </property>
    <widget class="QMenu" name="menuLayer">
     <property name="title">
      <string>Layer</string>
     </property>
     <addaction name="actionMixed_2"/>
     <addaction name="separator"/>
     <addaction name="actionFractal_1"/>
     <addaction name="actionFractal_2"/>
     <addaction name="actionFractal_3"/>
     <addaction name="actionFractal_4"/>
     <addaction name="separator"/>
     <addaction name="actionPicture_1"/>
     <addaction name="actionPicture_2"/>
     <addaction name="separator"/>
     <addaction name="actionWatermark"/>
    </widget>
    <widget class="QMenu" name="menuImage_quiality">
     <property name="title">
      <string>Image quiality</string>
     </property>
     <addaction name="actionHigh"/>
     <addaction name="actionLow"/>
    </widget>
    <widget class="QMenu" name="menuCoordinate_mouse_tracking">
     <property name="title">
      <string>Coordinate mouse tracking</string>
     </property>
     <addaction name="actionOn"/>
     <addaction name="actionOff"/>
    </widget>
    <addaction name="menuLayer"/>
    <addaction name="menuImage_quiality"/>
    <addaction name="menuCoordinate_mouse_tracking"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionRoute_Tool_Help"/>
    <addaction name="separator"/>
    <addaction name="actionFractal_Designer_Help"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionBack_to_main_window">
   <property name="icon">
    <iconset resource="Icon_resources.qrc">
     <normaloff>:/Menu Icon/Previous.svg</normaloff>:/Menu Icon/Previous.svg</iconset>
   </property>
   <property name="text">
    <string>Back to main window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRoute_Tool_Help">
   <property name="icon">
    <iconset resource="Icon_resources.qrc">
     <normaloff>:/icon/Menu Icon/Help.svg</normaloff>:/icon/Menu Icon/Help.svg</iconset>
   </property>
   <property name="text">
    <string>Route Tool Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionMixed">
   <property name="text">
    <string>Mixed</string>
   </property>
  </action>
  <action name="actionFractal_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fractal 1</string>
   </property>
  </action>
  <action name="actionFractal_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fractal 2</string>
   </property>
  </action>
  <action name="actionFractal_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fractal 3</string>
   </property>
  </action>
  <action name="actionFractal_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fractal 4</string>
   </property>
  </action>
  <action name="actionMixed_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mixed</string>
   </property>
  </action>
  <action name="actionHigh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>High</string>
   </property>
  </action>
  <action name="actionLow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionFractal_Designer_Help">
   <property name="icon">
    <iconset resource="Icon_resources.qrc">
     <normaloff>:/icon/Menu Icon/Help.svg</normaloff>:/icon/Menu Icon/Help.svg</iconset>
   </property>
   <property name="text">
    <string>Fractal Designer Help</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
   <property name="shortcut">
    <string>Shift+V</string>
   </property>
  </action>
  <action name="actionSave_Point">
   <property name="icon">
    <iconset resource="Icon_resources.qrc">
     <normaloff>:/icon/Menu Icon/Save.svg</normaloff>:/icon/Menu Icon/Save.svg</iconset>
   </property>
   <property name="text">
    <string>Save Point</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPicture_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Picture 1</string>
   </property>
  </action>
  <action name="actionPicture_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Picture 2</string>
   </property>
  </action>
  <action name="actionWatermark">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Watermark</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icon_resources.qrc"/>
 </resources>
 <connections/>
</ui>
